health-services/plan-service/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this module will be documented in this file.
 
+
+## 1.0.1 - 2025-01-30
+1. Mixed Distribution Strategy for Registration and Service Delivery
```

Comment:
This should be reworded to added/enabled support for mixed distribution strategy for registration and service delivery.